daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Runner.scala

```diff
@@ -225,14 +218,8 @@ object Runner {
 
     }
     for {
-      // Apparently Future.traverse is too stupid to traverse an
-      // Option so you first have to convert to a list and then convert back.
-      defClient <- Future
-        .traverse(participantParams.default_participant.toList)(client(_))
-        .map(_.headOption)
-      otherClients <- Future
-        .traverse(participantParams.participants)({ case (k, v) => client(v).map(c => (k, c)) })
-        .map(_.toMap)
+      defClient <- participantParams.default_participant.traverse(client(_))
```

Comment:
what happened with `headOption` it is not required any longer.... `participantParams.default_participant` is `Option[A]`, right?

Reply:
Indeed.